packages/apptive_grid_form/pubspec.yaml

```diff
@@ -8,8 +8,15 @@ environment:
   sdk: '>=2.17.0 <3.0.0'
   flutter: ">=1.17.0"
 
+dependency_overrides:
+  apptive_grid_core:
+    git:
+      url: https://github.com/ApptiveGrid/ApptiveGrid-flutter
+      path: packages/apptive_grid_core
+      ref: 70b87e5
+
```

Comment:
This will make this package not publishable!

Reply:
I'll wait for the 0.13.0 release then

Reply:
If you run `melos boostrap` it should've resolved itself anyways?

Reply:
done